theano/gpuarray/type.py

```diff
@@ -470,7 +471,17 @@ def c_headers(self):
                 '<gpuarray_api.h>']
 
     def c_header_dirs(self):
-        return [pygpu.get_include(), numpy.get_include()]
+        other_dirs = []
+        alt_inc_dir = os.path.abspath(os.path.normpath(sys.exec_prefix + '/Library/include'))
```

Comment:
@lamblin the `/Library` path seems to exist only with Conda distribution and only on Windows, so it is more a specific fix for Conda on Windows.

There are indeed `include` and `lib` paths on Linux, but I thought it was correctly included by Theano, maybe not after all ... I'll make some tests on Ubuntu tomorrow.